app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="signup-cont">
```

Comment:
- Kindly follow the design guidelines for the Signup page:
   - [ ] Kindly add the `Back link` instead of the hamburger menu which redirects to the previous page.
   - [ ] Please add the `Next link` instead of the `Sign out` link in the navbar in the header responsible for the sign-up process instead of adding a signup button.
   - [ ] Kindly remove the input's label and use placeholders instead.

| Your design | Guidelines |
| --- | --- |
| ![image](https://github.com/Christelle-12/Budget-manager/assets/24703171/af18d5a5-0808-4583-9799-307ac284eb30) | ![image](https://user-images.githubusercontent.com/24703171/219884879-31b72a01-5fe0-4116-8fee-3a397873ebb9.png) |